bin/curriculum/copy_level_concept_difficulties_between_years.rb

```diff
@@ -12,6 +16,10 @@ def main(year_to_copy_to)
       puts "Level: " + sl.level.name
       next if sl.level.level_concept_difficulty
       copy_lcd_from_parent(sl.level)
+
+      # Write changes to file
+      file_path = Level.level_file_path(sl.level.name)
+      File.write(file_path, sl.level.to_xml)
```

Comment:
Heads up @dmcavoy that this caused the problem addressed in #40742. Can you own follow up? it looks like you would just need to check that the level is `.custom?` here, or possibly that `level_num == 'custom'` (unfortunately these are slightly different).

Reply:
I'm going to just remove the script in the update to copy LCDs when we clone. Working on it here https://github.com/code-dot-org/code-dot-org/pull/40751